packages/loaders/test/fetch.test.js

```diff
@@ -0,0 +1,26 @@
+import { fetch } from '../dist/index.js';
```

Comment:
I am a bit conflicted here... calling `fetch` like the global could confuse unless the users check the top of the file to discover that it's imported. Is it the standard of the community?